.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,28 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+    - name: Lint with flake8
+      run: |
+        pip install flake8
```

Comment:
We should create a requirements.txt file for pip to use so we don't end up with a huge list in travis. This will be easier for the team to keep up to date in their environments anyways. I think intellij suite actually can detect out of date requirements files too.

Reply:
Right now, we don't have it. It can be updated later.